test/jdk/java/lang/Math/PowTests.java

```diff
@@ -57,6 +57,21 @@ static int testNonstrictPowCase(double input1, double input2, double expected) {
         return failures;
     }
 
+    static int testStrictVsNonstrictPowCase(double input1, double input2) {
+        double smResult = StrictMath.pow(input1, input2);
+        double mResult = Math.pow(input1, input2);
+        int failures = 0;
+        if (Double.isNaN(smResult)) {
+            if (!Double.isNaN(mResult)) failures = 1;
+        } else {
+            failures += Tests.testUlpDiff(
+                "StrictMath.pow(double, double) vs Math.pow(double, double)",
+                input1, input2, mResult, smResult, 1.0
```

Comment:
In theory at least, one pow method could round an ulp up and the other could round an ulp down so the difference should be 2 ulps (ignoring the special case of values straddling a power of two).

Reply:
Good point.